nipype/interfaces/freesurfer/utils.py

```diff
@@ -954,6 +954,63 @@ def _gen_outfilename(self):
         return name + ext + "_converted." + self.inputs.out_datatype
 
 
+class MRIsCombineInputSpec(FSTraitedSpec):
+    """
+    Uses Freesurfer's mris_convert to combine two surface files into one.
+    """
+    in_file1 = File(exists=True, mandatory=True, position=1,
+                    argstr='--combinesurfs %s',
+                    desc='File to be combined with in_file2')
+    in_file2 = File(exists=True, mandatory=True, position=2,
+                    argstr='%s',
+                    desc='File to be combined with in_file1')
```

Comment:
I didn't realize `traits.List` allowed for `maxlen` and `minlen`. I definitely prefer it your way. Just tested it and it works, so I'll commit it in a second.